components/Author/Tabs/FeedCard.tsx

```diff
@@ -64,7 +64,7 @@ export type FeedCardProps = {
   created_by: any;
   document: any;
   created_date: any;
-  documentFilter: any;
+  documentFilterdocumentFilter: any;
```

Comment:
Was this change intended?

Reply:
Good catch @gzurowski probably some keyboard error